py-polars/polars/internals/construction.py

```diff
@@ -140,7 +140,7 @@ def arrow_to_pyseries(name: str, values: pa.Array, rechunk: bool = True) -> PySe
         elif array.num_chunks == 0:
             pys = PySeries.from_arrow(name, pa.array([], array.type))
         else:
-            pys = PySeries.from_arrow(name, array.combine_chunks())
+            pys = PySeries.from_arrow(name, array.chunks[0])
```

Comment:
I think we should do this only if `array.num_chunks == 1` end leave the `else` branch as is.

Reply:
I'm happy to do that, but just to make sure we're in the same page, the existing if has next conditions:

- `array.num_chunks > 1`
- `array.num_chunks == 0`
- `else`

Assuming `num_chunks` will always be an integer number greater or equal to zero, adding `array.num_chunks == 1` will make that the else code is never executed.

Another option worth considering is having `array.num_chunks == 0` and `array.num_chunks == 1` and then have the rest as `else` instead of the current `array.num_chunks > 1`.

As said, happy to implement what you say, but just want to make sure we're understanding things the same way. Thanks

Reply:
Oh, sorry, my bad. 

I didn't expand the lines above. You are right. :+1: